package.json

```diff
@@ -108,5 +108,8 @@
     "ts-node": "^10.9.2",
     "typescript": "^5.4.5",
     "typescript-eslint": "^7.8.0"
+  },
+  "dependencies": {
+    "cross-fetch": "^4.0.0"
```

Comment:
Why do we need this?  All modern browsers now implement the fetch function natively, and we don't need a polyfill because this code never runs in Node...

An important aspect of userale is that we have 0 dependencies. This is a major selling point to USG users.

Reply:
This was a a remnant of me trying to test getting fetch to work in the test environment. Does Jest not run in Node? Anyway I removing this.